tests/integration/extension/src/background.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable no-unused-vars */
```

Comment:
We can probably remove this since you are using webScience below?

Reply:
Yes this was from the older test extension that didn't do anything except test the build (which this also does), thanks!